streampark-flink/streampark-flink-submit/streampark-flink-submit-core/src/main/scala/org/apache/streampark/flink/submit/trait/FlinkSubmitTrait.scala

```diff
@@ -356,8 +358,12 @@ trait FlinkSubmitTrait extends Logger {
 
     val programArgs = new ArrayBuffer[String]()
 
-    Try(submitRequest.args.split("\\s+")).getOrElse(Array()).foreach(x => if (x.nonEmpty) programArgs += x)
-
+    //    Try(submitRequest.args.split("\\s+")).getOrElse(Array()).foreach(x => if (x.nonEmpty) programArgs += x)
+    val pattern = Pattern.compile("[^\\s\"']+|\"[^\"]*\"|'[^']*'")
```

Comment:
The Reg-exp Pattern  need to be defined in advance. Please refer to line 34 of streampark-flink-submit-api/FlinkSubmitter